pubspec.yaml

```diff
@@ -40,6 +40,9 @@ dependencies:
   otp_text_field: ^1.1.3
   flutter_secure_storage: ^8.1.0
   http: ^0.13.5
+  get_it: ^8.2.0
```

Comment:
use this create ine dependency injection class add all the dependencies register the blocs usecases